source/code-generation.txt

```diff
@@ -0,0 +1,56 @@
+.. _rm-code-generation:
+
+===============
+Code Generation
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Relational Migrator can assist you in converting both your SQL and 
+application code to MongoDB. By utilizing Relational Migrator's code 
+generation features, you can streamline the migration process and 
+save time and effort.
+
+Use Cases
+---------
+
+You can use the following features within the 
+:guilabel:`Code Generation` tab to convert your legacy code:
+
+.. list-table::
+   :header-rows: 1
+   
+   * - Feature 
+     - Supported Conversions 
+
+   * - Query Converter 
+     - 
+      - SQL queries 
+      - SQL views
+      - Stored procedures 
+   * - Generate App Code
+     - 
+      - ``C#``, ``Java``, ``JavaScript``, and ``JSON`` application code.
+
+
+.. note::
+
+   Query Converter is currently in private preview. For sign-up details, 
+   see :ref:`rm-query-converter`.
+
+Get Started
+-----------
+
+- :ref:`<rm-generate-app-code>`
+- :ref:`<rm-query-converter>`
```

Comment:
[suggestion]

Can we change the wording in the link so it reads as a task? Having a concept under Get Started reads a bit strangely.

Reply:
Sure I updated to:
```
Convert your application code
Convert your SQL code
```